h/api/nipsa/logic.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+from h.api.nipsa import models
+
+
+def _publish(request, data):
+    request.get_queue_writer().publish("nipsa_user_requests", data)
+
+
+def index():
+    """Return a list of all the NIPSA'd user IDs.
+
+    :rtype: list of unicode strings
+
+    """
+    return [nipsa_user.userid for nipsa_user in models.NipsaUser.all()]
+
+
+def nipsa(request, userid):
+    """NIPSA a user.
+
+    Add the given user's ID to the list of NIPSA'd user IDs.
+    If the user is already NIPSA'd then nothing will happen (but a "nipsa"
+    message for the user will still be published to the queue).
+
+    """
+    nipsa_user = models.NipsaUser.get_by_id(userid)
+    if not nipsa_user:
```

Comment:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=True/False_evaluations#True/False_evaluations

The reason for using `if nipsa_user is None:` seems to be in case `nipsa_user` is some falsey value other than `None`, for example in case of a method that may return either `None` or an empty list,  but in this case I know that `NipsaUser` objects aren't falsey and don't expect `get_by_id()` to return any falsey values except `None` so I think _Use the "implicit" false if at all possible_ applies.